atomicapp/utils.py

```diff
@@ -360,13 +360,17 @@ def getUniqueUUID():
         return data
 
     @staticmethod
-    def loadAnswers(answers_file):
+    def loadAnswers(answers_file, format=None):
         if not os.path.isfile(answers_file):
             raise AtomicAppUtilsException(
                 "Provided answers file does not exist: %s" % answers_file)
 
         logger.debug("Loading answers from file: %s", answers_file)
-        return anymarkup.parse_file(answers_file)
+        try:
+            result = anymarkup.parse_file(answers_file, format=format)
+        except anymarkup.AnyMarkupError:
+            result = anymarkup.parse_file(answers_file)
+        return result
```

Comment:
Actions:
- I will add some inline docs
- I will check the INI file format to see if it allows ``:`` in the section name